src/Livewire/Settings/TicketTypeEdit.php

```diff
@@ -43,10 +47,15 @@ public function mount(): void
             null
         );
 
-        $appModels = get_subclasses_of(Model::class, 'FluxErp\\');
-        $moduleModels = get_subclasses_of(Model::class, 'Modules\\');
+        $fluxModels = get_subclasses_of(Model::class, 'FluxErp\\Models\\');
```

Comment:
better use modelfinder from data tables for this as in the SearchBar component.
Should be more performant because its cached.